ui/test-view/redux/store.js

```diff
@@ -0,0 +1,88 @@
+import configureStore from './configureStore';
+
+export const { store, actions } = configureStore();
+// Map platform key to a user-friendly platform name
+export const platformMap = {
```

Comment:
I think this can now be removed and `values.js` used directly?

Reply:
Ahh, great point.   Thanks!